.cirrus.yml

```diff
@@ -78,8 +78,8 @@ task:
     HOME: /root
     CIRRUS_WORKING_DIR: /home/runc
     GO_VERSION: "1.23"
-    BATS_VERSION: "v1.9.0"
-    RPMS: gcc git iptables jq glibc-static libseccomp-devel make criu fuse-sshfs container-selinux
+    BATS_VERSION: "v1.11.0"
+    RPMS: cpio gcc git iptables jq qemu-kvm glibc-static libseccomp-devel make criu fuse-sshfs container-selinux
```

Comment:
Probably the QEMU test should be executed only on a single GHA Ubuntu job

Reply:
We could do that, but the test itself runs quite quickly (1.6s on my machine) so I don't think it's worth making the running more complicated.